nipype/interfaces/freesurfer/petsurfer.py

```diff
@@ -0,0 +1,651 @@
+        outputs = self.output_spec().get()
+        # Get the top-level output directory
+        if not isdefined(self.inputs.pvc_dir):
+            pvcdir = os.getcwd()
+        else:
+            pvcdir = os.path.abspath(self.inputs.pvc_dir)
+        outputs["pvc_dir"] = pvcdir
+
+        # Assign the output files that always get created
+        outputs["ref_file"] = os.path.join(pvcdir, "km.ref.tac.dat")
+        outputs["hb_nifti"] = os.path.join(pvcdir, "km.hb.tac.nii.gz")
+        outputs["hb_dat"] = os.path.join(pvcdir, "km.hb.tac.dat")
+        outputs["nopvc_file"] = os.path.join(pvcdir, "nopvc.nii.gz")
+        outputs["gtm_file"] = os.path.join(pvcdir, "gtm.nii.gz")
+        outputs["gtm_stats"] = os.path.join(pvcdir, "gtm.stats.dat")
+        outputs["reg_pet2anat"] = os.path.join(pvcdir, "aux/bbpet2anat.lta")
+        
+        # Assign the conditional outputs
+        if isdefined(self.inputs.save_input) and self.inputs.save_input:
+            outputs["input_file"] = os.path.join(pvcdir, "input.nii.gz")
+        if isdefined(self.inputs.save_yhat0) and self.inputs.save_yhat0:
+            outputs["yhat0"] = os.path.join(pvcdir, "yhat0.nii.gz")     
+        if isdefined(self.inputs.save_yhat) and self.inputs.save_yhat:
+            outputs["yhat"] = os.path.join(pvcdir, "yhat.nii.gz")  
+        if isdefined(self.inputs.save_yhat_full_fov) and self.inputs.save_yhat_full_fov:
+            outputs["yhat_full_fov"] = os.path.join(pvcdir, "yhat_full_fov.nii.gz")
+        if isdefined(self.inputs.save_yhat_with_noise) and self.inputs.save_yhat_with_noise:
+            outputs["yhat_with_noise"] = os.path.join(pvcdir, "yhat_with_noise.nii.gz") 
+        if isdefined(self.inputs.mgx) and self.inputs.mgx:
+            outputs["mgx_ctxgm"] = os.path.join(pvcdir, "mgx.ctxgm.nii.gz")
+            outputs["mgx_subctxgm"] = os.path.join(pvcdir, "mgx.subctxgm.nii.gz")
+            outputs["mgx_gm"] = os.path.join(pvcdir, "mgx.gm.nii.gz")
+        if isdefined(self.inputs.rbv) and self.inputs.rbv:
+            outputs["rbv"] = os.path.join(pvcdir, "rbv.nii.gz")
+        if isdefined(self.inputs.opt) and self.inputs.opt:
+            outputs["opt_params"] = os.path.join(pvcdir, "aux/opt.params.dat")
+
+        return outputs
+
+    def _format_arg(self, name, spec, value):       
+        return super(GTMPVC, self)._format_arg(name, spec, value)
+
+class MRTMInputSpec(GLMFitInputSpec):
+
+    mrtm1 = InputMultiPath(
+        traits.Tuple(File(exists=True, mandatory=True), File(exists=True, mandatory=True)),
+        argstr="--mrtm1 %s %s...",
+        desc="RefTac TimeSec : perform MRTM1 kinetic modeling",
+    )
+
+class MRTMOutputSpec(GLMFitInputSpec):
+
+    k2p = File(desc="estimate of k2p parameter")
+
+class MRTM(GLMFit):
+    """Perform MRTM1 kinetic modeling.
+    Examples
+    --------
+    >>> mrtm = MRTM()
+    >>> mrtm.inputs.in_file = 'tac.nii'
+    >>> mrtm.inputs.mrtm1 = [('ref_tac.dat', 'timing.dat')]
```

Comment:
These files should just be empty files in `nipype/testing/data` to make the tests pass.